hack/arktos-cni.rc

```diff
@@ -66,6 +69,23 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+install_cni_mizar() {
+    echo "Ensuring firewall to allow traffic forward by default"
+    sudo iptables -S FORWARD | grep '\-P' | grep DROP && sudo iptables -P FORWARD ACCEPT
+    sudo iptables -S FORWARD | grep '\-P'
+
+    export FEATURE_GATES="AllAlpha=false,MandatoryArktosNetwork=true"
+    export ARKTOS_NETWORK_TEMPLATE=mizar
+
+    ensure_cni_binaries
```

Comment:
Please investigate why the single-yaml doesn't do this.

Reply:
These environment variables are available in the arktos only and used during deployment, at mizar don't have these variables

Reply:
What environment variables?

Reply:
When we deploy artkos cluster, we need these environment variables to Enforce the Multi-tenant Network Feature.
We are following the steps from this [guide](https://github.com/CentaurusInfra/arktos/blob/master/docs/setup-guide/arktos-enforces-network-feature.md) and automating steps here in the script.

Reply:
Are you referring to FEATURE_GATES and ARKTOS_NETWORK_TEMPLATE env variables? I don't have a problem with that. My question was regarding why we need to call ensure_cni_binaries. What does it do for Mizar?

Reply:
And following the guide, why don't we need ```export ARKTOS_NO_CNI_PREINSTALLED=y``` but need the other two?

Reply:
The code is removed for `ensure_cni_binaries`, as it is not required. Thanks.